winui/AvatarView/Content-Type.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post 
+title: Content Types in WinUI AvatarView control | Syncfusion
+description: Learn what are the content types in AvatarType and how to set value for the content types in WinUI AvatarView.
+platform: WinUI
+control: SfAvatarView
+documentation: ug
+---
+
```

Comment:
@MadhumithanJayapathi Content types in WinUI Avatar View

Reply:
@Manivannan-E Changed the title as mentioned.